eng/scripts/Create-APIView.ps1

```diff
@@ -29,17 +32,61 @@ function Get-SwaggerReadMeFile {
       }
 
       $readmeFile = Get-ChildItem -Path $currentPath -Filter "readme.md" -File -ErrorAction SilentlyContinue
-      if ($readmeFile -and $readmeFile.Name -eq "readme.md") {
+      if ($readmeFile -and $readmeFile.Name -eq "readme.md" -and (Assert-AutoRestConfiguration -readMeFile $readmeFile.FullName)) {
           return $readmeFile.FullName
```

Comment:
I would suggest just trying to parse the readme and return the parsed object here if it is valid if not go to the next one. That will prevent use parsing here in assert in the get tags function.